src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,107 @@
+package duke;
+
+import duke.TaskList;
+import duke.ToDo;
+import duke.Deadline;
+import duke.Event;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Objects;
+import java.util.Scanner;
```

Comment:
I like the way you organised your imports so it is neater and easier to read, good job!